HW_module_4_task2.py

```diff
@@ -0,0 +1,59 @@
+import string
+import random
+
+# create a list of random number of dicts (from 2 to 10)
+def create_list_of_dict(a, b, c=4, e=5, f=1):
```

Comment:
a, b, c, d? Imagine that you will return to this code after a month. How will you understand what c is? You will need to review all functions. The arguments must also have meaningful names.